CANserver/Logging.cpp

```diff
@@ -290,6 +320,54 @@ void CANServer::Logging::handleMessage(CAN_FRAME *frame, const uint8_t busId)
                     break;
                 }
 
+                case CANServer::Logging::LogType_Drive:
+                {
+                    long currentLogMillis = millis();
+                    int gearNUM = 0;
+                    String tempRow = String(_currentTimeOfDay.tv_sec);
+                    CANServer::CanBus *canbusInstance = CANServer::CanBus::instance();
+                    CANServer::CanBus::AnalysisItemMap::const_iterator found_it = canbusInstance->dynamicAnalysisItems()->find("Gear");
+                    gearNUM = found_it->second->lastValue;
```

Comment:
Need to check to see if the gear analysis item isn't found.

if (found_it != canbusInstance->dynamicAnalysisItems()->end())

Otherwise I think this will crash